torchvision/prototype/transforms/_utils.py

```diff
@@ -22,7 +22,7 @@ def get_chw(image: Union[PIL.Image.Image, torch.Tensor, features.Image]) -> Tupl
     if isinstance(image, features.Image):
         channels = image.num_channels
         height, width = image.image_size
-    elif isinstance(image, torch.Tensor):
+    elif isinstance(image, torch.Tensor):  # datumbox: should this have been is_simple_tensor() instead?
```

Comment:
cc @vfdev-5 / @pmeier

Reply:
it is only used in `query_chw` where types are already prefiltered:
```python
    chws = {
        get_chw(item)
        for item in flat_sample
        if isinstance(item, (features.Image, PIL.Image.Image)) or is_simple_tensor(item)
    }
```

Reply:
You are right, the code is valid and works as expected due to the prior filtering. I would argue that it might be easier to read if we use the `is_simple_tensor()` idiom here. WDYT?

Reply:
https://github.com/pytorch/vision/pull/6498